package.json

```diff
@@ -4,13 +4,13 @@
   "description": "Use this to save yourself some time when working on a webpack configuration.",
   "main": "index.js",
   "scripts": {
-    "cover": "nyc --reporter=lcov --reporter=text npm t",
+    "cover": "nyc --reporter=lcov --reporter=text ava --coverage",
```

Comment:
>> Have you tried: `"cover": "nyc --reporter=lcov --reporter=text ava"`?

Thats a good catch!! It works, So nothing to do with passing `--coverage` but **not use `npm test`**